lib/features/align-elements/AlignElementsContextPadProvider.js

```diff
@@ -70,7 +70,7 @@ AlignElementsContextPadProvider.prototype._getEntries = function() {
     'align-elements': {
       group: 'align-elements',
       title: self._translate('Align elements'),
-      imageUrl: ICONS['align'],
+      html: `<div class="entry">${ICONS['align']}</div>`,
```

Comment:
Shouldn't it be this?
```suggestion
      imageHtml: `<div class="entry">${ICONS['align']}</div>`,
```

Reply:
Actually, we used `imageHtml` to allow html for popup menu entries since the icon is just one element making up the entry. With context pad, this was already in place with `html`: https://github.com/bpmn-io/diagram-js/blob/develop/lib/features/context-pad/ContextPad.js#LL320C32-L320C36 - we did not change this. Also for context pad, it represents the whole component since there is only the icon.

We could decide to change it to make it more consistent but would introduce a breaking change there.

Reply:
OK now I get it. Thanks for the explanation.